service/controller/key/key.go

```diff
@@ -31,6 +31,22 @@ const (
 	ProviderOperatorKVM   = "kvm-operator"
 )
 
+func AppConfigDeployed(app applicationv1alpha1.App, configs corev1alpha1.ConfigList) bool {
+	for _, config := range configs.Items {
+		_, configManagedByReleaseOperator := config.Labels[LabelManagedBy]
```

Comment:
Again this only check the annotation existence. It would be good compare the value to `project.Name()` IMO.

Reply:
ah, true